server/src/shared/interfaces/StatsUserTimestamp.ts

```diff
@@ -0,0 +1,6 @@
+import { Types } from 'mongoose';
+
+export type StatsUserTimestamp = Array<{
```

Comment:
wouldn't it be better to define the `export class` with `@Prop()` as it is for the rest schemas (like for example `base.object.schema` etc.)?

Reply:
Not sure what you mean tbh